tools/msvs/msi/nodemsi.wixproj

```diff
@@ -14,25 +14,25 @@
     <NodeVersion Condition=" '$(NodeVersion)' == '' ">0.0.0.0</NodeVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
-    <OutputPath>..\..\..\$(Configuration)\</OutputPath>
```

Comment:
I did it on OSX too and this matches what happens on Linux and elsewhere, the main reason for this is to make it easier to define artifacts for Jenkins to upload. Currently we only need to put `iojs-${version}*.*` in Jenkins for it to pick everything up but without the changes in this PR it relies on some scripting in the Jenkins build process to move the files out of the subdirectories.

Can we move it out for now so it's consistent across all platforms and then move back if it ever becomes necessary? I reckon if it becomes necessary to distribute debug builds then we put `dbg` or similar in the filename so it's properly disambiguated.